charts/ccd-data-store-api/values.preview.template.yaml

```diff
@@ -9,6 +9,7 @@ java:
         - data-store-api-draft-key
         - ccd-data-s2s-secret
   environment:
+    AZURE_APPLICATIONINSIGHTS_INSTRUMENTATIONKEY: "2dcb834e-768e-4429-9050-ab15af959995"
```

Comment:
Testing purpose only. Will remove / comment it out later

Reply:
confirmed this is that of a new, temporary ai resource.